onnxruntime/core/providers/openvino/openvino_execution_provider.cc

```diff
@@ -127,6 +125,7 @@ bool IsOpSupported(std::string name, std::string device) {
       "Relu",
       "Reshape",
       "Shape",
+      "Selu",
```

Comment:
Can you please add ops in alphabetical order, these ops are arranged alphabetically for easier readability.

Reply:
Done